Makefile

```diff
@@ -135,10 +138,16 @@ BUILD = GOOS=$(1) GOARCH=$(2) \
 
 # BUILD_TARGETS is the set of all platforms and architectures that Git LFS is
 # built for.
-BUILD_TARGETS = bin/git-lfs-darwin-amd64 bin/git-lfs-darwin-386 \
-	bin/git-lfs-linux-amd64 bin/git-lfs-linux-386 \
-	bin/git-lfs-freebsd-amd64 bin/git-lfs-freebsd-386 \
-	bin/git-lfs-windows-amd64.exe bin/git-lfs-windows-386.exe
+BUILD_TARGETS = \
```

Comment:
If you plan to make the ARM targets part of the default set of build targets (i.e., the ones that are usually built with `make build` or `make release`), I think that those should be added here, too.

The ones that I see that are missing are:

- `bin/git-lfs-darwin-arm64`
- `bin/git-lfs-freebsd-arm64`
- `bin/git-lfs-windows-arm64.exe`

Reply:
GO doesn't support arm64 on those OSes. I added them in for future go, but can just drop if you prefer.

Reply:
In that case, let's drop the ones that are missing here and below from the Makefile entirely.

Reply:
done